api/controllers/kubernetes/install/app_test.go

```diff
@@ -1,112 +0,0 @@
-package install
-
-import (
-	"context"
-	"testing"
-
-	"github.com/replicatedhq/embedded-cluster/pkg/release"
-	kotsv1beta1 "github.com/replicatedhq/kotskinds/apis/kots/v1beta1"
-	"github.com/replicatedhq/kotskinds/multitype"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func TestInstallController_GetAppConfig(t *testing.T) {
-	tests := []struct {
-		name           string
-		releaseData    *release.ReleaseData
-		expectedConfig kotsv1beta1.Config
-		expectedError  bool
-	}{
```

Comment:
I get that right now the controller isn't doing much besides proxying requests to the manager but I would imagine that soon enough that might not be the case(?) E.g. doing the validation that we're currently doing in the handler, or the state transitions. Should we keep this test file around even if the only tests we have are 2 with the manager mocked where it errors if the manager returns an error and returns the manager response otherwise?

No strong feelings about it though.